README.md

```
@@ -131,16 +150,20 @@ following structure:
 
-#### `identity_roles_accessible`
+#### `identity_roles_accessible` (via mixin)
+
+> **Note:** This feature has been moved to a mixin in v2.0.0. To use it, vendor the `policy-Identity-role-TeamAccess.tf` mixin.
 
-The `identity_roles_accessible` element provides a list of role names corresponding to roles created in the
+The `aws_teams_accessible` variable (when using the mixin) provides a list of role names corresponding to roles created in the
 `iam-primary-roles` component. For each named role, a corresponding permission set will be created which allows the user
 to assume that role. The permission set name is generated in Terraform from the role name using a statement like this one:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)